bp/index.html

```diff
@@ -661,7 +662,16 @@ <h3>Data publication</h3>
 
         <p>Each of the four starting points outlined above have their own challenges, but working with plain text documents can be particularly tricky as you will need to parse the natural language to identify the <a>Spatial Things</a> and their properties before you can proceed any further. Natural Language Processing (NLP) is a complex topic in its own right and is beyond the scope of this best practice document. We will assume that you’ve already completed this step and have parsed any plain documents into structured data records of some kind.</p>
       </section>
-
+      <section id="scope-FAIR">
+	<h3>FAIR Principles</h3>
+	  <p>The FAIR Principles are described at <a href="https://www.go-fair.org/fair-principles/">FAIR Principles - GO FAIR</a>; they are widely adopted (or at least aimed for)
+	    when publishing scientific data including environmental and earth observation data. 
+	    Although the FAIR principles concentrate on machine readable data, whilst these best practices also cover “data for humans”,
+	    there is a lot of overlap between the FAIR Principles and the best practices described in this paper.</p>
+	  <p>Similarly, although not currently expressed in terms of the FAIR Principles,
+	    the Data on the Web Best Practices are also designed to make it easier for "data consumers to find, use and link to the data".</p>
+	  <p>There have also been some suggestions for improvement on the FAIR principles, and these are also discussed <a href="#FAIRPrinciples" class="sectionRef"></a>.</p>
```

Comment:
Would this refer to the section entitled "FAIR Challenges/extensions" (doesn't have an id)